### PR TITLE
S02 us4 twig formation

### PR DESCRIPTION
**Pull request by Erdnaxela45**

### CODE REVIEW

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

File: public/assets/css/formation.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    height: auto;
+    font-family: var(--font-family);
+    color: var(--primary-color);
+}
+
+.programme {
+    text-align: center;
+    font-size: var(--font-small-size);
+    margin-bottom: var(--margin-bottom-prestations);
+}
```
Comment (LucyMerlier) [inline public/assets/css/formation.css:13]: english?

File: src/View/Formation/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                <div class="col-12 col-sm-12 col-md-9 col-lg-9">
+                    <p>Les qualités requises pour mettre à profit son don</p>
+                    <p>Développer ses capactités extra-sensorielles (mise en pratique)</p>
+                    <p>Quel support utiliser</p>
+                    <p>Contacter ses guides spirituels (mise en pratique)</p>
+                    <p>Pourquoi sommes-nous fatigués? Comment se ressourcer?</p>
+                    <p>Comment se purifier des maux de la clientèle après une consultation (stress, tristesse, maux de tête,..)</p>
+                    <p>Qu'est-ce qui peut bloquer votre médiumnité?</p>
+                    <p>Comment fonctionnent les flashs ou visions? Comment les interpréter? (mise en pratique)</p>
+                </div>
```
Comment (LucyMerlier) [inline src/View/Formation/index.html.twig:27]: this very much looks like a list

File: src/View/Formation/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            </div>
+        </div>
+        <div class="part row d-flex align-items-start justify-content-around p-5">
+            <h3>Les protections</h3>
+            <div class="col-12 col-sm-12 col-md-3 col-lg-3">
+                <img src="../assets/images/protection.jpg" class="img-fluid" alt="">
```
Comment (LucyMerlier) [inline src/View/Formation/index.html.twig:33]: alt attribute is mandatory

File: src/View/Formation/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            </div>
+            <div class="col-12 col-sm-12 col-md-9 col-lg-9 ">
+                <p>La préparation avant et pendant la consultation</p>
+                <p>L'encrage (mise en pratique)</p>
+                <p>Lithothérapie : connaissance des pierres principales pour son activité</p>
+            </div>
```
Comment (LucyMerlier) [inline src/View/Formation/index.html.twig:39]: this looks like a list

File: src/View/Formation/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        </div>
+        <div class="part row d-flex align-items-start justify-content-around p-5">
+            <h3>Les esprits</h3>
+            <div class="col-12 col-sm-12 col-md-3 col-lg-3">
+                <img src="../assets/images/esprits.jpg" class="img-fluid" alt="">
```
Comment (LucyMerlier) [inline src/View/Formation/index.html.twig:44]: alt attribute is mandatory

File: src/View/Formation/index.html.twig [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+            </div>
+            <div class="col-12 col-sm-12 col-md-9 col-lg-9 ">
+                <p>Que sont les esprits</p>
+                <p>Les différents esprits</p>
+                <p>Pourquoi s'en défaire</p>
+                <p>Comment se purifier</p>
+                <p>Les symptômes d'un esprit accroché à soi</p>
+            </div>
```
Comment (LucyMerlier) [inline src/View/Formation/index.html.twig:52]: list?

File: src/View/Formation/index.html.twig [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+        </div>
+        <div class="part row d-flex align-items-start justify-content-around p-5">
+            <h3>Les magies</h3>
+            <div class="col-12 col-sm-12 col-md-3 col-lg-3">
+                <img src="../assets/images/magie.jpg" class="img-fluid" alt="">
```
Comment (LucyMerlier) [inline src/View/Formation/index.html.twig:57]: alt?

File: src/View/Formation/index.html.twig [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+            </div>
+            <div class="col-12 col-sm-12 col-md-9 col-lg-9 ">
+                <p>La magie noire, comment s'en protéger?</p>
+                <p>La magie blanche, votre meilleure alliée</p>
+            </div>
```
Comment (LucyMerlier) [inline src/View/Formation/index.html.twig:62]: list?

File: src/View/Formation/index.html.twig [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+        </div>
+            <div class="part row d-flex align-items-start justify-content-around p-5">
+            <h3>Culture générale</h3>
+            <div class="col-12 col-sm-12 col-md-3 col-lg-3">
+                <img src="../assets/images/culture.jpg" class="img-fluid" alt="">
```
Comment (LucyMerlier) [inline src/View/Formation/index.html.twig:67]: alt?

File: src/View/Home/_prestations.html.twig [hunk 10]
```diff
@@ -6,13 +6,27 @@
...
+                <div class="card" style="width: 18rem;">
+                    <div class="card-header">
+                        <h3>{{ prestation.name }}</h3>
+                    </div>
+                    <div class="card-body">
+                        <p class="card-text-center">{{ prestation.description }}</p>
+                    </div>
+                    <ul class="list-group list-group-flush">
+                        <li class="list-group-item">Tarif 1 heure : {{ prestation.price1hour }} €</li>
+                        <li class="list-group-item">Tarif 30 minutes : {{ prestation.price30min }} €</li>
```
Comment (LucyMerlier) [inline src/View/Home/_prestations.html.twig:19]: what if there is no 30min price?

File: src/View/Home/_prestations.html.twig [hunk 11]
```diff
@@ -6,13 +6,27 @@
...
+                    </ul>
+                </div>
+            {% endfor %}
             <div class="card" style="width: 18rem;">
```
Comment (LucyMerlier) [inline src/View/Home/_prestations.html.twig:23]: NO STYLE IN HTML

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments + it looks like there are 2 user stories here (training and services) + please use english even for class names, image names, adn folder names + all your images are too heavy

**Pull request closed without merging**